addons/account_peppol/views/res_partner_views.xml

```diff
@@ -32,6 +33,9 @@
                                     string="Verify"
                                     help="Verify partner's PEPPOL endpoint"/>
                         </div>
+                        <div class="alert alert-warning" role="alert" invisible="not account_peppol_is_endpoint_valid or bank_account_count != 0 and country_code">
+                            To generate electronic invoices, also set a country and a bank account for this partner.
```

Comment:
if it's a new term, it should be added to addons/account_peppol/i18n/account_peppol.pot